t/test-genome-DBs/homo_sapiens/variation/table.sql

```diff
@@ -325,7 +325,7 @@ CREATE TABLE `protein_function_predictions_attrib` (
 
 CREATE TABLE `publication` (
   `publication_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
-  `title` varchar(255) DEFAULT NULL,
+  `title` varchar(400) DEFAULT NULL,
```

Comment:
@at7 @dglemos In the variation schema, this is a varchar(300), not 400. https://github.com/Ensembl/ensembl-variation/blob/be5c085fbb8f829562b866d1414b6fe5d9f949b5/sql/table.sql#L1741

Don't think it matters for a test DB patch, but wanted to point out the discrepancy in case it indicates an issue with the patcher.

Reply:
@ens-bwalts the PR for this patch wasn't merged yet. We are waiting for final confirmation from production.